common-app-lib/src/cmr/common_app/site/data.clj

```diff
@@ -41,4 +42,5 @@
    (merge (time-page)
           default-partner-guide
           {:base-url (config/application-public-root-url context)
-           :app-title app-title})))
+           :app-title app-title
+           :app-environment (config/app-environment)})))
```

Comment:
why not return the actual URL here instead of the environment?

Reply:
Done